erigon-lib/common/hash.go

```diff
@@ -61,6 +61,12 @@ func BigToHash(b *big.Int) Hash { return BytesToHash(b.Bytes()) }
 // If b is larger than len(h), b will be cropped from the left.
 func HexToHash(s string) Hash { return BytesToHash(hexutility.FromHex(s)) }
 
+func HexToRefHash(s string) *Hash {
```

Comment:
Rename to match the method above:
```suggestion
func HexToHashRef(s string) *Hash {
```